package.json

```diff
@@ -24,10 +26,8 @@
   },
   "scripts": {
     "test": "hardhat test",
-    "watch": "npm-watch",
-    "lint": "solium --dir ./contracts",
-    "prepublishOnly": "truffle compile && truffle networks --clean",
-    "pub": "npm run prepublishOnly && npm publish"
+    "deploy": "hardhat compile && hardhat run scripts/deploy.js",
+    "lint": "solium --dir ./contracts"
```

Comment:
Linting gives 14 errors and 51 warnings. For example it says `Syntax error: unexpected token a`, but everything compiles and tests run, so maybe solium is using the wrong solc version? Not sure if this is worth spending time on now

Reply:
Hmmm yeah it's barfing on Solidity stuff, i.e. the abstract keyword, so it must be linting for the wrong version. Will make an issue to look into it.